islandora.module

```diff
@@ -1685,8 +1690,12 @@ function islandora_object_access($op, $object, $user = NULL) {
  */
 function islandora_islandora_object_access($op, $object, $user) {
   module_load_include('inc', 'islandora', 'includes/utilities');
-
-  return islandora_namespace_accessible($object->id) && user_access($op, $user);
+  if ($object->state != 'A') {
+    return islandora_namespace_accessible($object->id) && user_access($op, $user) && user_access(ISLANDORA_ACCESS_INACTIVE_AND_DELETED_OBJECTS, $user);
```

Comment:
@DiegoPino That would definitely handle the new user scenario.  I'd have to test to confirm it would work when importing a feature.

@fuel37 Are there any features DGI has developed for clients that are publicly available and susceptible to the proposed changes?  It would be great if we could test with something from the wild and confirm that there would be no impact to your team.